scripts/just.config.ts

```diff
@@ -6,7 +6,7 @@ export const types = tscTask({ emitDeclarationOnly: true });
 
 export const build = () =>
   esbuildTask({
-    entryPoints: glob.sync("src/**/*.ts"),
+    entryPoints: glob.sync("src/**/*.{ts,tsx}"),
```

Comment:
you technically did make changes in /scripts which has a package.json in the "workspace" - flip that package.json to have private flag would stop that noise.

Reply:
Yes, very true, my bad. I feel like I keep having to do it for this package but raised it at the wrong time. Earlier on there was a mode change from 644 to 755 iirc.

Reply:
Added `private: true` 👍